microprofile/microprofile-opentracing-1.3/src/main/java/org/wildfly/swarm/ts/microprofile/opentracing/v13/interfaces/OnMethod.java

```diff
@@ -0,0 +1,61 @@
+package org.wildfly.swarm.ts.microprofile.opentracing.v13.interfaces;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+import org.wildfly.swarm.ts.microprofile.opentracing.v13.TracedService;
+
+@Path("/interface/onmethod")
+public class OnMethod {
```

Comment:
Again, why an extra endpoint for the client? Couldn't all clients point to the same resource?